plugins/modules/cloud/profitbricks/profitbricks.py

```diff
@@ -583,7 +583,7 @@ def main():
             volume_size=dict(type='int', default=10),
             disk_type=dict(choices=['HDD', 'SSD'], default='HDD'),
             image_password=dict(default=None, no_log=True),
-            ssh_keys=dict(type='list', elements='str', default=[]),
+            ssh_keys=dict(type='list', elements='str', default=[], no_log=False),
```

Comment:
Is this correct? The dict has the private key as a key:https://github.com/ansible-collections/community.general/blob/3053a85e567fd5a70e6801b464f04f9b5b266672/plugins/modules/remote_management/manageiq/manageiq_provider.py#L174
ir does this not get logged?

Reply:
`auth_key` itself is marked as `no_log=True`: https://github.com/ansible-collections/community.general/blob/3053a85e567fd5a70e6801b464f04f9b5b266672/plugins/modules/remote_management/manageiq/manageiq_provider.py#L593